Dockerfile-2.7

```diff
@@ -0,0 +1,47 @@
+FROM python:2.7-alpine3.6 
+
+LABEL maintainer="Arista Ansible Team <ansible@arista.com>"
+LABEL com.example.version="0.5"
+LABEL vendor1="Arista"
+LABEL com.example.release-date="2019-10-29"
+LABEL com.example.version.is-production="False"
+
+ENV PS1='py2.7-ansible-cvp:\u% '
+
+WORKDIR /tmp
+WORKDIR /tmp
+
+RUN apk add --update --no-cache ca-certificates \
+                                openssh-client \
+                                build-base \
+                                gcc \
+                                g++ \
+                                make \
+                                python-dev \
+                                py-pip \
+                                libffi-dev \
+                                sshpass \
+                                libssl1.0 \
+                                openssl-dev
+
+COPY requirements.txt .
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN apk del -r --purge gcc make g++ &&\
+    rm -rf /source/* &&\
+    rm -rf /var/cache/apk/* &&\
+    rm -rf /tmp/*
+
+WORKDIR /project
+ADD arista/ arista/
+RUN cat arista/cvp/galaxy.yml
+RUN ansible-galaxy collection build --force arista/cvp && \
+    for i in *.tar.gz; do ansible-galaxy collection install $i -p /usr/share/ansible/collections; done
+
+WORKDIR /playbooks
+ADD examples/* ./
+RUN cp inventory.ini.example inventory.ini
+
+VOLUME /playbooks
+CMD ["/bin/sh"]
```

Comment:
better change that to 
`ENTRYPOINT ["tail", "-f", "/dev/null"]`

Reply:
Not sure to understand:

It seems this approach will keep this container up and to connect to it, it will require a `docker exec` with container name

Whereas CMD approach allows user to spin-up container, attach shell for testing and automatically delete it when user close session.

As we are not exposing a service from this container, should we consider to get an always up approach ?

Reply:
actually, _i think_ it will do things a bit differently. what you have here in CMD will get overridden by the argument you pass in exec command. the reason why you'd want to have an entrypoint like this is if you want to have container running all the time. If it's not the case (as it is in our case), we can simply drop this CMD instruction, since it doesn't do anything.

Reply:
Ok understood. Will remove this CMB block